exec/java-exec/src/main/java/org/apache/drill/exec/store/easy/json/JSONRecordReader.java

```diff
@@ -64,6 +64,8 @@
   private final boolean enableAllTextMode;
   private final boolean readNumbersAsDouble;
   private final boolean unionEnabled;
+  private int parseErrorCount;
```

Comment:
This should be long instead of int since parseErrorCount is cumulative..so in the worst case it could be as large as runningRecordCount.